custom_components/hacs/manifest.json

```
@@ -18,8 +18,9 @@
     "aiofiles>=0.5.0",
     "aiogithubapi>=2.0.0<3.0.0",
     "backoff>=1.10.0",
+    "fnvhash==0.1.0",
```

```suggestion
    "fnvhash>=0.1.0<1.0.0",
```
@bdraco Should probably make this more forgiving incase it updates in core?